torch/_weights_only_unpickler.py

```diff
@@ -356,10 +356,17 @@ def load(self):
                         "``torch.distributed.tensor`` must be imported to load DTensors"
                     )
                 else:
+                    if "builtins" in full_path and "builtins" == full_path[:8]:
+                        full_path = full_path[9:]
+                        full_path = (
+                            full_path[1:]
+                            if len(full_path) > 0 and full_path[0] == "."
```

Comment:
nit:

if len(full_path) > 0 should it be guaranteed that full_path[0] == "." here, either that or we should not have removed the builtins (?)